src/librustdoc/html/static/main.js

```diff
@@ -1832,6 +1832,15 @@ function getSearchElement() {
 
     window.initSearch = initSearch;
 
+    function htmlToText(html) {
+        if (html.indexOf("<") === -1) {
+            return html;
+        }
```

Comment:
What about entities (e.g. `&amp;`)

Reply:
They're not harmful since they can't be interpreted as html.